### PR TITLE
[QA-528] Adding analyzer property check and query test support for v312x, v311 and v310

### PR DESCRIPTION
**Pull request by MDAlFattah**

### CODE REVIEW

File: release_tester/selenium_ui_test/pages/analyzers_page.py [hunk 1]
```diff
@@ -1112,6 +1114,397 @@ def add_new_analyzer(self, name, ui_data_dir=None):
...
+                    expected_properties = ''.join(str(self.generate_expected_properties(name, ui_data_dir)).split())
+                    # Assert that the copied text matches the expected text
+                    try:
+                        assert actual_properties == expected_properties, "Text does not match the expected text \n"
+                    except AssertionError as ex:
+                        print("actual_properties: ", actual_properties)
+                        print("expected_properties: ", expected_properties)
+                        raise AssertionError("Text does not match the expected text") from ex
+                    else:
+                        print("Text matches the expected text. \n")
```
Comment (alexbakharew) [inline release_tester/selenium_ui_test/pages/analyzers_page.py:1154]: It is better to rewrite this check:
```
if actual_properties != expected_properties:
    print("actual_properties: ", actual_properties)
    print("expected_properties: ", expected_properties)
    raise Error("Properties are not equal for analyzer " + analyzer.name())
```

File: release_tester/selenium_ui_test/pages/analyzers_page.py [hunk 2]
```diff
@@ -1112,6 +1114,397 @@ def add_new_analyzer(self, name, ui_data_dir=None):
...
+
+            except TimeoutException as ex:
+                print(f'Failed to parse properties from the {name} and the error is: {ex} \n')
```
Comment (alexbakharew) [inline release_tester/selenium_ui_test/pages/analyzers_page.py:1157]: Why we catch and do not throw again? Potentially, it will hide errors from us

**Review by alexbakharew — CHANGES_REQUESTED** [review]
Please rewrite check for properties

File: release_tester/selenium_ui_test/pages/analyzers_page.py (hunk 1 above)
Comment (MDAlFattah) [reply release_tester/selenium_ui_test/pages/analyzers_page.py]: fixed.

File: release_tester/selenium_ui_test/pages/analyzers_page.py (hunk 2 above)
Comment (MDAlFattah) [reply release_tester/selenium_ui_test/pages/analyzers_page.py]: fixed.

**Review by alexbakharew — CHANGES_REQUESTED** [review]
this check should be executed for all supported versions: 3.10, 3.11, 3.12

**Review by alexbakharew — APPROVED** [review]
LGTM

**Pull request merged**